frontend/src/styles.scss

```diff
@@ -54,3 +55,17 @@ input:-webkit-autofill:active {
   width: 100%;
 }
 
+body::-webkit-scrollbar {
+  width: 1em;
+	background-color: rgba($color: #1a0c0c3d, $alpha: 0.2)
+}
+
+ body::-webkit-scrollbar-track {
+  background: linear-gradient(90deg,#434343,#434343 1px,#111 0,#111);
+ }
+
+
+body::-webkit-scrollbar-thumb {
+  background: #434343;
+  box-shadow: inset 2px 2px 2px hsl(0deg 0% 100% / 25%), inset -2px -2px 2px rgb(0 0 0 / 25%);
+}
```

Comment:
Does it work for you?
In my case scrollbar's design hasn't changed. I think you don't need `body` here